app/_hub/kong-inc/response-transformer/index.md

```diff
@@ -69,44 +65,56 @@ params:
       required: false
-      description: List of original_header_name:new_header_name pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `original_header_name:new_header_name` pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
     - name: replace.headers
       required: false
-      description: List of headername:value pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `headername:value` pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
     - name: replace.json
       required: false
-      description: List of property:value pairs. If and only if the parameter is already present, replace its old value with the new one. Ignored if the parameter is not already present.
+      datatype: array of string elements
+      description: List of `property:value` pairs. If and only if the parameter is already present, replace its old value with the new one. Ignored if the parameter is not already present.
     - name: replace.json_types
       required: false
+      datatype: array of string, boolean, or number elements
       description: List of JSON type names. Specify the types of the JSON values returned when replacing JSON properties.
     - name: add.headers
       required: false
       value_in_examples: ["x-new-header:value","x-another-header:something"]
-      description: List of headername:value pairs. If and only if the header is not already set, set a new header with the given value. Ignored if the header is already set.
+      datatype: array of string elements
+      description: List of `headername:value` pairs. If and only if the header is not already set, set a new header with the given value. Ignored if the header is already set.
     - name: add.json
       required: false
       value_in_examples: ["new-json-key:some_value", "another-json-key:some_value"]
-      description: List of property:value pairs. If and only if the property is not present, add a new property with the given value to the JSON body. Ignored if the property is already present.
+      datatype: array of string elements
+      description: List of `property:value` pairs. If and only if the property is not present, add a new property with the given value to the JSON body. Ignored if the property is already present.
     - name: add.json_types
       required: false
-      value_in_examples: ["new-json-key:string", "another-json-key:number"]
+      value_in_examples: ["new-json-key:string", "another-json-key:boolean", "another-json-key:number"]
+      datatype: array of string, boolean, or number elements
```

Comment:
same question as https://github.com/Kong/docs.konghq.com/pull/2604/files#r580949567 applies here

Reply:
done in commit 49fb609